### ISSUE TITLE
Reload config file problem

### ISSUE THREAD

**Issue by javisank** [nielsen-oss/fasttext-serving#4]
Hi, I have been using tensorflow serving for a few months, and now I have to serve fasttext models.
I like very much the feature on TF Serving that you can reload models, updating your config file. I could not find my way with this on fasttext-serving.

Here is my code to try and reload the config file, to serve a second model. 
```
import os
import grpc

from fts.protos import service_pb2, service_pb2_grpc

if __name__ == "__main__":

    # Generate GRPC stub
    channel = grpc.insecure_channel("localhost:50051")
    stub = service_pb2_grpc.FastTextStub(channel)

    request = service_pb2.ReloadModelsRequest()
    response = stub.ReloadConfigModels(request)
    print(response)
    request = service_pb2.LoadModelsRequest()
    response = stub.GetLoadedModels(request)
    print(response)
    loaded_models = [model.name for model in response.models]
    print(loaded_models)
```

I am trying with "yelp_review_polarity" and "dbpedia". Here my config.yaml file. I tried loading the server with just "yelp_review_polarity", then adding "dbpedia" on config.yaml, and sending a request from the previous client code.

```
# List of models to serve
models_path: sample/models
models:
  - base_path: yelp_review_polarity
    name: yelp_review_polarity
  - base_path: dbpedia
    name: dbpedia
```

What am i doing wrong?

**Comment by defr8001** [comment]
Hi @javisank ,

We have merged a hot fix for this issue in the PR #6. Can you pull the latest changes and see if it works?

Regards

**Comment by defr8001** [comment]
Hi @javisank,

Did it work for you? Can we close this issue?
